docs/usage/dataset.rst

```diff
@@ -16,14 +16,15 @@ Create an Epitome Dataset
 
 First, create an Epitome Dataset. In the dataset, you will define the
 ChIP-seq targets you want to predict, the cell types you want to train from,
-and the assays you want to use to compute cell type similarity.
+the assays you want to use to compute cell type similarity, and the data directory
+to where you configured your pre-processed data.
 
 .. code:: python
 
  	targets = ['CTCF','RAD21','SMC3']
 	celltypes = ['K562', 'A549', 'GM12878']
 
-	dataset = EpitomeDataset(targets=targets, cells=celltypes)
+	dataset = EpitomeDataset(targets=targets, cells=celltypes, data_dir="configured_data_path")
```

Comment:
can you add a comment of what `configured_data_path` is above this line? Can you change `configured_data_path` to `path/to/configured/data`

Reply:
Done